control-plane/roles/monitoring/tasks/main.yaml

```diff
@@ -40,6 +41,7 @@
     - monitoring.coreos.com_prometheusrules.yaml
     - monitoring.coreos.com_servicemonitors.yaml
     - monitoring.coreos.com_thanosrulers.yaml
+  when: monitoring_crds_enabled
```

Comment:
Same here. Should this not rather be: `monitoring_prometheus_operator_crds_enabled`